process/best_practices_badge.md

```diff
@@ -0,0 +1,57 @@
+
+This document outlines specific questions Academy Software Foundation projects have raised in regard to fulfilling the badge requirements. If you have questions not answered, please provide a PR that adds your question below and will be merged with an answer.
+
+## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+
+Having a primary developer take the three courses outlined below ( all free to audit ) would be sufficient:
+
+- [Secure Software Development: Requirements, Design, and Reuse (LFD104x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-requirements-design-and-reuse-lfd104/) 
+- [Secure Software Development: Implementation (LFD105x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-implementation-lfd105/) 
+- [Secure Software Development: Verification and More Specialized Topics (LFD106x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-verification-and-more-specialized-topics-lfd106/) 
+
+## Is there a list or document which describes the static analysis tools that ASWF supports or recommends?
+
+SonarCloud is the tool that multiple projects have implemented, and there is already an organization-level access token that projects can use with it. You can see a specific example in the OpenEXR analysis_workflow.yml, which shows how to build, run the test suite, post-process, and upload to SonarCloud using the SONAR_TOKEN from the organization level. You can see the results for the various ASWF projects with SonarCloud here:
+
+[https://sonarcloud.io/organizations/academysoftwarefoundation/projects](https://www.google.com/url?q=https://sonarcloud.io/organizations/academysoftwarefoundation/projects&sa=D&source=docs&ust=1675375752427760&usg=AOvVaw3aJ1DsMv9yF2Xf-Uhv8EKo)
+
+I'm also seeing some work towards using clang-tidy in Imath: [https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake#L118](https://www.google.com/url?q=https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake%23L118&sa=D&source=docs&ust=1675375752432129&usg=AOvVaw1HThXkwYas6U7fU6_IKetY)
+
```

Comment:
A more robust link would be:

https://github.com/AcademySoftwareFoundation/Imath/blob/main/config/ImathSetup.cmake